### PR TITLE
Movies project - Marjaana Tienaho

### PR DESCRIPTION
**Pull request by MayaBlixt**
Link to my deployed project:
https://marjaanasmovies.netlify.app/

My reflections on how this week's project turned out:

Things I'd like to have clarified or explained in more detail:

### CODE REVIEW

File: code/public/index.html [hunk 1]
```diff
@@ -4,31 +4,20 @@
...
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+    <meta name="description" content="Top movies list ">
+    <meta name="keywords" content="Top movies, movie list, movie information">
+    <meta name="author" content="Marjaana Tienaho">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="icon" type="image/png" sizes="32x32" href="/favicon-32x32.png">
```
Comment (LillanEriksson) [inline code/public/index.html:12]: Really nice favicon and great that you added og:tags

File: code/src/App.js [hunk 2]
```diff
@@ -1,9 +1,29 @@
...
+    <BrowserRouter>
+      <Switch>
+        <Route path="/"exact>
+          <MovieList />
+        </Route>
+        <Route path="/movies/:slug" exact>
+          <MovieDetails />
+        </Route>
+        <Route path="/404" >
+          <NotFound />
```
Comment (LillanEriksson) [inline code/src/App.js:21]: This route does not seem to work, I did not manage to get to it, it just displayed netlifys 404 not found page. It did not work in my project either, so guess we'll have to look into why it works on our local machines, but not  when deployed.

**Review by LillanEriksson — CHANGES_REQUESTED** [review]
Nice work Marjaana! It looks great! Clean and readable. Would you just fix the 404 not found-page? :)

File: code/src/components/MovieCard.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    title, 
+    id,
+    poster_path,
+    release_date,
+    slug
+    }) =>{
+
+    return( 
+        /* Display movie image, title and releasedate */
+        <section className='movie-card'>
```
Comment (EmOlsz) [inline code/src/components/MovieCard.js:15]: There are some wrong indentations in this components

File: code/src/pages/MovieDetails.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
+import React, { useState, useEffect} from 'react'
+import { useParams, Link } from 'react-router-dom'
+
+import BackArrow from '../components/icons8-back-arrow-64.png';
```
Comment (EmOlsz) [inline code/src/pages/MovieDetails.js:4]: Uppercase naming is reserved for components only, not images. it should rather be :

```
import backArrow from '../components/icons8-back-arrow-64.png';
```

File: code/src/pages/NotFound.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+import NotFoundImg from '../components/404.jpg';
```
Comment (EmOlsz) [inline code/src/pages/NotFound.js:3]: Same here

**Pull request closed without merging**